relay-general/src/store/normalize.rs

```diff
@@ -254,6 +255,33 @@ impl<'a> NormalizeProcessor<'a> {
     }
 }
 
+fn normalize_app_start_spans(event: &mut Event) {
+    if !event.sdk_name().eq("sentry.javascript.react-native")
+        || !(event.sdk_version().starts_with("4.4")
+            || event.sdk_version().starts_with("4.3")
+            || event.sdk_version().starts_with("4.2")
+            || event.sdk_version().starts_with("4.1")
+            || event.sdk_version().starts_with("4.0")
+            || event.sdk_version().starts_with('3'))
```

Comment:
Just to double check, these are only the effected versions you would like to normalize?

Reply:
Yes. Only these.